gef-ui/server/routes/supporting-information-upload.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const multer = require('multer');
+const { multerFilter } = require('../utils/multer-filter.utils');
+const {
+  uploadSupportingDocument, deleteSupportingDocument,
+} = require('../controllers/supporting-information/supporting-documents');
+const { validateRole, validateToken, validateBank } = require('../middleware');
+
+// The following routes cannot use the same csrf checks as the rest of the routes
+// as the Ministry of Justice multi-file-upload component does not allow passing tokens to the request body.
+// So we instead make a separate uploadCsrf token and check it here.
+const router = express.Router();
+
+// eslint-disable-next-line consistent-return
+const validateUploadCsrfToken = (req, res, next) => {
+  if (req.session.uploadCsrf
+    && req.session.uploadCsrf.token === req.query.uploadCsrf
+    && new Date() < new Date(req.session.uploadCsrf.expiry)) {
```

Comment:
`<=`, any thoughts?

Reply:
We could use date-fns' `isBefore` function for this too?

Reply:
When doing comparisons between two date objects javascript automatically converts them to their epoch time behind the scenes to do the comparison so it's doing this for us for free :)